mycli/main.py

```diff
@@ -688,15 +691,36 @@ def output(self, output, status=None):
 
         """
         if output:
-            self.log_output(output)
-            special.write_tee(output)
-            special.write_once(output)
-
-            if (self.explicit_pager or
-                    (special.is_pager_enabled() and not self.output_fits_on_screen(output, status))):
-                click.echo_via_pager(output)
-            else:
-                click.secho(output)
+            size = self.cli.output.get_size()
+
+            margin = self.get_reserved_space() + self.get_prompt(self.prompt_format).count('\n') + 1
```

Comment:
I think I accidentally removed `output_fits_on_screen` when I was resolving the merge conflicts, thanks for noticing, I'll try to fix it as soon as possible 😄